### PR TITLE
add copied files of pre-course project

### PR DESCRIPTION
**Pull request by tiagomarin**
Files and images were copied from a project I did on the pre-course in order to apply a code review to my own code. This is just a practice to acquire the necessary skills to give good code reviews.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
```
Comment (tiagomarin) [inline index.html:5]: This line should start with the same indentation from the previous one. Remember to follow the rules and to apply the linters. This way your code will be easier to read for you and others.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            <form id="search-box" action="http://thecodeblock.com">
+                <input  type="text" placeholder="Search..">
+                <button  type="submit" value="search">&nbsp;</button>
+            </form>
+            <div>
+                
+            </div>
+        </nav>
+    </header>
+
```
Comment (tiagomarin) [inline index.html:31]: You shouldn't leave empty spaces in your code. Try using the indentation to keep your code organized and visible.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+        </div>
+        
+        <div class="icon-text">
+            
+                <img class="icons" src="images/like.png" alt="Like Button">&nbsp;Yeah!&nbsp;&nbsp;>
+                <img class="icons" src="images/dislike.png"" alt="Dislike Button">&nbsp;   Nah!&nbsp;&nbsp;>
+            
+        </div> 
+        <div>
+            <h3>
```
Comment (tiagomarin) [inline index.html:50]: This seems like a text, so please use the correct tag for that purpose. Check this guide out if you need a reminder on tags:
https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+<link href="https://fonts.googleapis.com/css2?family=Quicksand:wght@300;500&display=swap" rel="stylesheet">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>TIAGOtube</title>
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    </head>
+<body>
+    <header>
+        <nav id= "nag-bar">
```
Comment (tiagomarin) [inline index.html:15]: It looks like the id "nag-bar" was supposed to be "nav-bar". Correct it if that's the case, if it is the way you intended, please disregard this comment.

**Review by tiagomarin — COMMENTED** [review]
Good job so far!

image

The videos are loading and everything on the website looks to be in place!

There are only a few improvements that could be done to make your code look more professional. 

Follow the comments below:

**Review by ishimwezachee — COMMENTED** [review]
Hi @Tiago-Lelinski-Marin , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**